internal/cmd/providers.go

```diff
@@ -190,10 +240,13 @@ $ infra providers add okta --url example.okta.com --client-id 0oa3sz06o6do0muoW5
 	cmd.Flags().StringVar(&opts.ClientID, "client-id", "", "OIDC client ID")
 	cmd.Flags().StringVar(&opts.ClientSecret, "client-secret", "", "OIDC client secret")
 	cmd.Flags().StringVar(&opts.Kind, "kind", "oidc", "The identity provider kind. One of 'oidc, okta, azure, or google'")
+	cmd.Flags().StringVar(&opts.ProviderAPIOptions.PrivateKey, "private-key", "", "The private key used to make authenticated requests to Google's API")
+	cmd.Flags().StringVar(&opts.ProviderAPIOptions.ClientEmail, "client-email", "", "The email assigned to the Infra service client in Google")
+	cmd.Flags().StringVar(&opts.ProviderAPIOptions.DomainAdmin, "domain-admin", "", "The email of your Google workspace domain admin")
```

Comment:
Will these always be pretty specific to google? Maybe we should name the flags `--google-private-key`, `--google-client-email`, etc?

Reply:
I changed to `--service-account--key` and `--service-account--email` from Jeff's suggestion in a different comment. These will be Google specific as far as Im aware at this time though.